doc/examples/transform/plot_swirl.py

```diff
@@ -72,9 +72,10 @@ def shift_left(xy):
 
 
 image = data.checkerboard()
-swirled = swirl(image, rotation=0, strength=10, radius=120, order=2)
+swirled = swirl(image, rotation=0, strength=10, radius=120)
 
-fig, (ax0, ax1) = plt.subplots(1, 2, figsize=(8, 3), sharex=True, sharey=True, subplot_kw={'adjustable':'box-forced'})
+fig, (ax0, ax1) = plt.subplots(1, 2, figsize=(8, 3), sharex=True, sharey=True,
```

Comment:
Either this notation or the one new from `plot_adapt_rgb.py` (with `nrows` and `ncols` explicitly assigned).